posthtml.config.js

```diff
@@ -0,0 +1,14 @@
+const dotenv = require('dotenv')
+
+dotenv.config()
+
+module.exports = {
+  plugins: {
+    'posthtml-expressions': {
+      locals: {
+        APP_TITLE: process.env.REACT_APP_TITLE ?? 'Oasis Explorer',
+        APP_DESCRIPTION: process.env.REACT_APP_DESC ?? 'Official explorer for the Oasis Network.',
+      },
+    },
+  },
+}
```

Comment:
Don't see a reason why we need a default here. And I would remove `posthtml-expressions` dependency.

Reply:
> Don't see a reason why we need a default here. 

The original idea was that all of these values in `.env` are optional, only required for re-branding, but everything works properly even without them.

We have changed that thinking now, so these values are required, so the defaults are no longer needed, so I have removed them.

Reply:
> I would remove `posthtml-expressions` dependency.

What method would you suggest for writing the required meta tags in index.html?

The original implementation used `React-Helmet`, but @buberdds suggested doing it statically at compile time.

As far as I know, with parcel, the way to use env variables in html at compile times is the `posthtml-expressions` plugin. Do you have another suggestion?

Reply:
> Could you try `@plasmohq/parcel-transformer-inject-env`

Yes, that works, too. Replaced `posthtml-expression` with that one.